.solhint.json

```diff
@@ -1,18 +1,14 @@
 {
   "extends": "solhint:recommended",
   "rules": {
-    "compiler-version": ["off"],
-    "constructor-syntax": "warn",
-    "quotes": ["error", "single"],
+    "compiler-version": ["warn"],
+    "quotes": "off",
     "func-visibility": ["warn", { "ignoreConstructors": true }],
-    "not-rely-on-time": "off",
```

Comment:
@0xteddybear why removing this rule ser?

Reply:
it's not necessary to explicitly disable it since it was removed from solhint:recommended